will/backends/io_adapters/base.py

```diff
@@ -40,6 +40,11 @@ def normalize_incoming_event(self, event):
 
     def handle_incoming_event(self, event):
         try:
+            user_id = event.get('user', 'wHATEver')
```

Comment:
Probably just being thick but the need for this constant isn't obvious to me, what's it for?

Reply:
That's me not trusting that every possible incoming event will actually have a 'user' element.

Reply:
Er, sorry should have been more specific. Does setting it to `wHATEver` when the user element is missing do something different than allowing it to be `None` ?

Reply:
... nope.  Sigh.

Reply:
Gotcha, just wanted to make sure I wasn't missing something. Thanks.

Reply:
I think `user` is missing when it's a bot or webhook. As the intent here is to just do a list lookup can we go with an empty string instead?